lib/components/viewers/stop-viewer.js

```diff
@@ -166,17 +168,20 @@ class StopViewer extends Component {
               ? <span>{stopData.name}</span>
               : <span>Loading Stop...</span>
             }
-            <Button
-              onClick={this._toggleFavorite}
-              bsSize='large'
-              style={{
-                color: this._isFavorite() ? 'yellow' : 'black',
-                padding: 0,
-                marginLeft: '5px'
-              }}
-              bsStyle='link'>
-              <Icon type={this._isFavorite() ? 'star' : 'star-o'} />
-            </Button>
+            {showUserSettings
+              ? <Button
```

Comment:
You can just do `showUserSettings && <Button...` instead of using a ternary.

Reply:
Yea, I'm aware -- I was just kind of feeling the ternary here. Does this stand out as a major style concern to you? I use both, but I did like something I read recently that called out using ternaries in favor of `&&` because it was more readable. Also, this: https://github.com/airbnb/javascript/issues/520#issuecomment-167003046

>>> The [`condition && <Button...`] example looks cleanest, but it suffers from a footgun: if condition is zero, it will be toString'ed and rendered.

Reply:
Ah, that reminds me of https://stackoverflow.com/a/43375944/269834. Maybe if we care a lot about this, we can add a linting rule or something.